db/extract.go

```diff
@@ -141,7 +141,7 @@ ON CONFLICT DO NOTHING;`
 }
 
 func (batch *Batch) UpdateISCNHeight(height int64) {
-	batch.Batch.Queue(`UPDATE meta SET height = $1 WHERE id = 'iscn'`, height)
+	batch.Batch.Queue(`UPDATE meta SET height = $1 WHERE id = 'extractor'`, height)
```

Comment:
do we need a new extractor when we add new feature in the future? in this case please add suffix like `extractor_v1`